keyboards/cest73/tkm/rules.mk

```diff
@@ -7,15 +7,12 @@ BOOTLOADER = halfkay
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = no        # Mouse keys
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

Reply:
Mousekey_enable should be "yes" and bootmagic_enable should be "lite" 
(minimal boot magic only)
I have no clue why yes or is that some kind of obsolete parameter?

Reply:
Bootmagic Full has been removed, the only options are now `yes` and `no`.

Reply:
What fauxpark said.  We've removed the "full" bootmagic options since basically all it did was cause issues.  

For anyone that absolutely wants/needs it, you can re-implement with bootmagic lite, actually.  

So, "yes" does what "lite" used to.